content/millennium/r4/care-provision/care-team.md

```diff
@@ -0,0 +1,140 @@
+---
+title: CareTeam | R4 API
+---
+
+# CareTeam
+
+* TOC
+{:toc}
+
+## Overview
+
+The R4 CareTeam resource can be used to represent care teams.
```

Comment:
I see that we have this sentence on CarePlan as well, but what information is it really communicating?

Reply:
Removed.